src/swagger/trips.swagger.js

```diff
@@ -294,3 +294,27 @@
 *       '400':
 *             description: Please double checks your departure and arrival dates.
 * */
+
+/**
+ * @swagger
+ * /api/v1/trips/requests/view:
```

Comment:
We could just remove `requests/` since we are not using it.

Reply:
I've worked on it.